python/GafferDispatchTest/ExecuteApplicationTest.py

```diff
@@ -352,7 +352,7 @@ def testCanSerialiseFrameDependentPlugs( self ) :
 
 		def validate( sequence ) :
 
-			s["PythonCommand"]["sequence"].setValue( sequence )
+			s["PythonCommand"]["framesMode"].setValue( GafferDispatch.PythonCommand.FramesMode.Sequence )
```

Comment:
Should we be setting `"framesMode"` based on the value of `sequence` here? Otherwise we're validating the same behaviour twice.

Reply:
Doh! Thanks for spotting that. I've squashed the fix into https://github.com/GafferHQ/gaffer/pull/6069/commits/af68f5d28e2518cfbf8c6161c70f8c7f124d016c.